### PR TITLE
Update ko_kr.json

### PR DESCRIPTION
**Pull request by Kardane**
Yet Another Translation Update

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/ko_kr.json [hunk 1]
```diff
@@ -212,18 +240,25 @@
...
+  "farmersdelight.item.skillet.how_to_cook": "다른 손에 재료를 드세요!",
 
   "farmersdelight.jei.cooking": "조리",
   "farmersdelight.jei.cutting": "도마",
+  "farmersdelight.jei.chance": "%1$s%% 확률",
   "farmersdelight.jei.decomposition": "분해",
   "farmersdelight.jei.decomposition.light": "햇빛에 의해 촉진됨",
   "farmersdelight.jei.decomposition.fluid": "근처 수분에 의해 촉진됨",
   "farmersdelight.jei.decomposition.accelerators": "근처 활성화제에 의해 촉진됨 (아래 참조)",
   "farmersdelight.jei.info.straw": "조합에 사용되는 지푸라기입니다. 잔디나 작물을 식칼로 잘라내 얻을 수 있습니다."
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/ko_kr.json]: There's a comma missing in this line, which will break JSON parsing.

**Comment by vectorwing** [comment]
Thanks for the update! (y)

I spotted a small JSON formatting problem, but otherwise, everything looks good.

**Comment by Kardane** [comment]
Oops :p

**Comment by vectorwing** [comment]
Thank you! ^^

**Pull request merged**